client/src/components/ChefCard.js

```diff
@@ -42,30 +49,82 @@ function ChefCard(props) {
-                            <Button
-                                key={cuisine}
-                                color="primary"
-                                variant="contained"
+                    <Grid
+                        container
+                        direction="column"
+                        justify="center"
+                        alignItems="center"
+                        spacing={2}
+                    >
+                        <Grid item>
+                            <Typography className={classes.name} gutterBottom>
+                                {firstName + " " + lastName}
+                            </Typography>
+                            <Typography className={classes.location} gutterBottom>
+                                {location}
+                            </Typography>
+                            <Typography className={classes.location} gutterBottom>
+                                {`${Math.round(distanceKm)} Km away`}
+                            </Typography>
+                        </Grid>
+                        <Grid
+                            item
+                            container
+                            direction="row"
+                            justify="center"
+                            alignItems="center"
+                        >
+                            {cuisineSpecialty &&
+                                cuisineSpecialty.map((cuisine) => (
+                                    <Button
+                                        key={cuisine}
+                                        color="primary"
+                                        variant="contained"
+                                    >
+                                        <Typography variant="button">
+                                            {cuisine}
+                                        </Typography>
+                                    </Button>
+                                ))}
+                        </Grid>
+                        <Grid item>
+                            <Typography className={classes.name}>{bio}</Typography>
+                        </Grid>
+                        <Grid item xs={12}>
+                            <Link
+                                component="button"
+                                variant="body2"
+                                className={classes.link}
+                                onClick={() =>
+                                    history.push({
+                                        pathname: "/chefprofile",
+                                        state: {
+                                            ...props.userId,
+                                            chefProfile: {
+                                                _id: props._id,
+                                                cuisineSpecialty:
+                                                    props.cuisineSpecialty,
```

Comment:
So by doing this instead of how we had it before, you won't be able to just visit a chef's profile based on the link. What if I wanted to send my housemate a link to the chef I'm considering booking for us? Was there a specific reason for this change?

Reply:
That is a very good point! I changed it because it was causing the menu drop down buttons to not work. When we discussed it during our standup this was the solution we came to. I'll bring your point up during standup today to see if the original way was better and should just fix the menu drop down.

Reply:
If you don't change it, this will be fine to merge without me looking at it again! Otherwise I want to take a look at the refactor

Reply:
We are going to leave it for now so we can focus on getting messaging working and made a new ticket to update in the future